src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -404,7 +417,7 @@ export default {
             }
         },
         containsObject(identifier) {
-            return _.get(this.domainObject, 'composition')
+            return this.domainObject.composition
```

Comment:
I think if we're replacing `_.get` we may want to check that composition exists (most likely will, but want to be safe), so we can put a check for this.domainObject.composition first and if it doesn't exist, just return;

Reply:
added a check